style.css

```diff
@@ -1006,3 +1093,1378 @@
+    width: 59px;
+    height: 20px;
+    font-family: var(--font4);
+    font-style: normal;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    color: #fff;
+    text-decoration: none;
+  }
+
+  .container-1 {
+    width: 100%;
+    min-height: 3455px;
+  }
+
+  .container-1 .section {
+    padding: 120px 24px 120px 24px;
+    width: 100%;
+    height: 590px;
+    background: #1c1a19;
+    background-image: url(images/background_img_desktop.png);
+    background-repeat: no-repeat;
+    background-position: bottom right;
+    display: flex;
+    flex-direction: column;
+    position: relative;
+  }
+
+  .first-page-intro {
+    margin-left: 89.01px;
+  }
+
+  .section h1 {
+    margin-right: 731px;
+    width: 327px;
+    height: 124px;
+    font-family: var(--font3);
+    font-style: normal;
+    font-weight: 400;
+    font-size: 56px;
+    line-height: 64px;
+    letter-spacing: 0.37px;
+    color: #ff6b00;
+  }
+
+  .section .title {
+    margin-right: 731px;
+    width: 327px;
+    height: 27px;
+    font-family: var(--font2);
+    font-style: normal;
+    font-weight: 700;
+    font-size: 24px;
+    line-height: 32px;
+    display: flex;
+    align-items: center;
+    color: #fff5e1;
+  }
+
+  .section .para {
+    width: 327px;
+    height: 123px;
+    font-family: var(--font1);
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    color: #fff;
+    margin-bottom: 12px;
+  }
+
+  .section ul {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 0;
+    width: 20px;
+    height: 120px;
+    top: 35%;
+    justify-content: space-between;
+    position: absolute;
+  }
+
+  .section ul li {
+    list-style: none;
+  }
+
+  .page2 {
+    height: 1805px;
+    width: 1204px;
+    position: relative;
+    margin: auto;
+  }
+
+  .desktop-page2 {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    width: 1133px;
+    height: 52px;
+    margin-left: 35.5px;
+    margin-right: 35.5px;
+    position: absolute;
+    top: 10%;
+  }
+
+  .main {
+    width: 1156px;
+    height: 1379px;
+    margin-top: 301px;
+  }
+
+  .div-page2 {
+    display: flex;
+    flex-direction: row;
+    width: 1156px;
+    height: 448px;
+    padding: 0;
+    gap: 24px;
+  }
+
+  .myworks {
+    width: 327px;
+    height: 83px;
+    font-family: var(--font3);
+    font-style: normal;
+    font-weight: 400;
+    font-size: 40px;
+    line-height: 52px;
+    display: flex;
+    align-items: center;
+    color: #1c1a19;
+  }
+
+  .line {
+    width: 327px;
+    height: 0;
+    border: 1px solid #212121;
+    margin-top: 24px;
+  }
+
+  .desktop-page2 .line {
+    width: 794.3px;
+    margin: auto;
+  }
+
+  .image-desktop {
+    height: 448px;
+    width: 684px;
+    background-image: url(images/Img_Placeholder_desktop_first.png);
+    background-repeat: no-repeat;
+  }
```

Comment:
- [ ] The same has happened here in the project cards. The placeholder image is not displayed and It may be that it doesn't exist in your project files
![image](https://user-images.githubusercontent.com/90222110/188198907-64e7579e-ca05-49d9-b7b4-0d153d7ba45c.png)